cpp/src/umap/supervised.h

```diff
@@ -249,17 +248,16 @@ void perform_general_intersection(const cumlHandle &handle, T *y,
   CUDA_CHECK(cudaPeekAtLastError());
 
   if (params->verbose) {
```

Comment:
Adding to my comment above, I think patterns like this would be removed altogether if we just set the log level at the beginning of the Python call into C++. These are debug logs and ideally we would only drop down to the DEBUG level when verbose=true. Otherwise, we could stay at INFO or WARN.